### PR TITLE
Add validaiton->validation, valuation

### PR DESCRIPTION
**Pull request by ydah**
8k files:
https://github.com/search?q=validaiton&type=code

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -31958,6 +31958,7 @@ invalaid->invalid
 invald->invalid
 invaldates->invalidates
 invalde->invalid
+invalidaiton->invalidation
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:31961]: Perhaps add plural too?
```suggestion
invalidaiton->invalidation
invalidaitons->invalidations
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -57519,6 +57520,8 @@ validadion->validation
 validadions->validations
 validador->validator
 validadors->validators
+validaiton->validation, valuation,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:57523]: Have you found actual cases where `validaiton` stands for `valuation`?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (ydah) [reply codespell_lib/data/dictionary.txt:31961]: Indeed it is. I updated.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (ydah) [reply codespell_lib/data/dictionary.txt:57523]: No. I have never seen a actual case, so I changed it to `validaiton->validation` only.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you @ydah.

**Pull request merged**